lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -214,21 +214,21 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
 
```

Comment:
I would have thought that we must already be hitting dark_lang on every request, so is it possible to cache it? We don't want to add an extra database call to every request just to support this feature.